### PR TITLE
Closes #56: Implemented the java bean for profile

### LINKED ISSUE #56: Implementing Java Beans
To make our webapp MVC compiant, Java Beans should be used to store information like the username, password, date or all sort of things.

### PR DESCRIPTION
**Pull request by Qw3rty26**
Closes #56, a new Java Class called `ProfileBean.java` was added inside the `/src/main/java` directory.

It implements the `java.io.Serializable` class and has everything needed to store the "name" and "password" attributes for the profile.

The constructor of the bean sets "name" and "password" to `null` by default, it is then needed to use the corrispective 
`void: setName(String: name)` and `void: setPassword(String: password)` 
methods to populate the bean.

After the Bean is successfully created and instantiated, we can then get "name" and "password" with the methods:

`String: getName(void)` and `String: getPassword(void)`

`Login.java` class has also been updated to correctly use `ProfileBean.java`.

### CODE REVIEW

File: src/main/java/ProfileBean.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	}
+
+
+
+
+	ProfileBean(){
+                name = null;
+                password = null;
+        }
+
```
Comment (San7o) [inline src/main/java/ProfileBean.java]: You should add the constructor
```java
ProfileBean(String name, String password){
    this.name = name;
    this.password = password;
}
```

**Review by San7o — CHANGES_REQUESTED** [review]
Add a method, everything else looks good

File: src/main/java/ProfileBean.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import java.io.Serializable;
+
+//The purpose of this Bean is to manage all the Profile's attributes 
+//(such as "name" or "password") within the webapp, whilst keeping it
+//MVC compliant and Thread safe.
+
+public class ProfileBean implements Serializable {
```
Comment (San7o) [inline src/main/java/ProfileBean.java]: Change `ProfileBean` to `TentativcAcessoBean`

File: src/main/java/Login.java [hunk 3]
```diff
@@ -81,20 +81,23 @@ protected void doPost (HttpServletRequest req,
         String url = "jdbc:derby://localhost:1527/DemoDB";
         Connection con = DriverManager.getConnection(url);
 
-        // Get parameters name and password
-        String name = req.getParameter("name");
-        String password = req.getParameter("password");
+        //Create Java Bean for profile
+	ProfileBean profile = new ProfileBean();
```
Comment (San7o) [inline src/main/java/Login.java]: change `ProfileBean` to `TentativoAccessoBean`

**Review by San7o — CHANGES_REQUESTED** [review]
Change the name of the class

**Comment by San7o** [comment]
I need this code fixes asap, I'll close the PR. This, with the fixes, ill be implemented in my next PR

**Comment by Qw3rty26** [comment]
@San7o I understand your immediate need of this Pull Request to be added to our webapp, however please note you can't just delete work already done just like that, instead please be a bit more patient and wait for changes to be made.

Thank you.

**Comment by Qw3rty26** [comment]
I updated the code for the Java Bean, its name is now `TentativoAccessoBean.java` as requested.
A new constructor was also added.
Everything should be looking good now.

**Comment by Qw3rty26** [comment]
this pull request isnt needed anymore after #72 so i guess it will be sent into the trash

**Pull request closed without merging**